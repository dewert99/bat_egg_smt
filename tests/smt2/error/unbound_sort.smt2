(declare-fun f () S1)