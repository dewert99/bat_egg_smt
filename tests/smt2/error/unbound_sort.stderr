the sort `S1` isn't bound a global constant at 1:19
(declare-fun f () S1)
                  ^

